buildSrc/build.gradle

```diff
@@ -19,3 +20,5 @@ dependencies {
 tasks.withType(Test).configureEach {
     useJUnitPlatform()
 }
+
+apply from: rootProject.layout.projectDirectory.file('../gradle/buildsrc-groovy-config.gradle')
```

Comment:
Do we need to extract this to it's own file?

Reply:
Done, moved to buildSrc/build.gradle.